network/src/web.ts

```diff
@@ -0,0 +1,114 @@
+import {
+  UnsupportedBrowserException,
+  WebPlugin,
+  PluginListenerHandle,
+} from '@capacitor/core';
+import {
+  NetworkPlugin,
+  NetworkStatus,
+  NetworkStatusConnectionType,
+} from './definitions';
+
+declare var window: any;
```

Comment:
We can be more specific about which types we're manipulating. We could go further and give `connection` a specific type as well, but there's only so many minutes in a day.

```suggestion
declare global {
  interface Navigator {
    connection: any;
    mozConnection: any;
    webkitConnection: any;
  }
}
```